stripe.go

```diff
@@ -101,12 +104,53 @@ type Backend interface {
 	SetMaxNetworkRetries(maxNetworkRetries int)
 }
 
-// BackendConfiguration is the internal implementation for making HTTP calls to Stripe.
+// BackendConfig is used to configure a new Stripe backend.
+type BackendConfig struct {
+	// HTTPClient is an HTTP client instance to use when maxing requests.
+	//
+	// If left unset, it'll be set to a default HTTP client for the package.
+	HTTPClient *http.Client
+
+	// LogLevel is the logging level of the library and defined by:
+	//
+	// 0: no logging
+	// 1: errors only
+	// 2: errors + informational (default)
+	// 3: errors + informational + debug
+	//
+	// Defaults to 0 (no logging), so please make sure to set this if you want
+	// to see logging output in your custom configuration.
+	LogLevel int
+
+	// Logger is where this backend will write its logs.
+	//
+	// If left unset, it'll be set to Logger.
+	Logger Printfer
+
+	// MaxNetworkRetries sets maximum number of times that the library will
+	// retry requests that appear to have failed due to an intermittent
+	// problem.
+	//
+	// Defaults to 0.
+	MaxNetworkRetries int
```

Comment:
Same as `LogLevel`.

Reply:
Thanks! Fixed that as well. (Its value was already on `BackendConfiguration`, but it was being carried over from the config struct.)